app/controllers/api/v1/events/nearests_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::Events::NearestsController < ApiController
+  def index
+    @events = Event.near(
+      [params[:lat], params[:lon]],
+      params[:radius],
+      units: :km
+    )
+    if @events.count(:all) > 0
```

Comment:
oh wait... you are getting a collection back,,,

You may want to do

`@event.any?`